recipes/ucc/meta.yaml

```
@@ -9,7 +9,7 @@ source:
     - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch  # [(cuda_compiler_version or "None").startswith("12")]
 
 build:
-  skip: True  # [not linux]
+  skip: True  # [(not linux) or (cuda_compiler_version == 'None')]
   number: 0
   missing_dso_whitelist:
```

Let's add an aggressive pinning for now:
```suggestion
  run_exports:
    - {{ pin_subpackage("ucc", max_pin="x.x.x") }}
  missing_dso_whitelist:
```